test/unit/middlewares/user-validator.test.js

```diff
@@ -128,5 +149,49 @@ describe("Middleware | Validators | User", function () {
       });
       expect(nextSpy.calledOnce).to.be.equal(false);
     });
+
+    it("Stops the propagation of the next if linkedin id is invalid", async function () {
+      const req = {
+        body: {
+          last_name: "patil",
+          first_name: "Abhay",
+          username: "invalidusername12",
+          twitter_id: "abhayisawesome",
+          linkedin_id: "@abhay2011",
+        },
+      };
+      const res = {
+        boom: {
+          badRequest: () => {},
+        },
+      };
+      const nextSpy = sinon.spy();
+      await updateUser(req, res, nextSpy).catch((err) => {
+        expect(err).to.be.an.instanceOf(Error);
+      });
+      expect(nextSpy.calledOnce).to.be.equal(false);
+    });
+
+    it("Stops the propagation of the next if linkedin id is invalid or contain spaces", async function () {
+      const req = {
+        body: {
+          last_name: "patil",
+          first_name: "Abhay",
+          username: "invalidusername12",
+          twitter_id: "abhayisawesome",
+          linkedin_id: "abhay 2011",
+        },
+      };
+      const res = {
+        boom: {
+          badRequest: () => {},
+        },
+      };
+      const nextSpy = sinon.spy();
+      await updateUser(req, res, nextSpy).catch((err) => {
+        expect(err).to.be.an.instanceOf(Error);
+      });
+      expect(nextSpy.calledOnce).to.be.equal(false);
+    });
```

Comment:
Nit: Can you write test for success also because I seen you written test for only for bad request
I am saying this point for better coverage

Reply:
@shubhamsigdar1 Written test for good-request as well, in both integration and unit test.